src/vite-plugin.ts

```diff
@@ -2,8 +2,9 @@ import type { TidewaveConfig } from './core';
 import { configureServer } from './http';
 import { getProjectName } from './core';
 import type { Plugin, ViteDevServer } from 'vite';
-// Import instrumentation to automatically patch console
-import './logger/instrumentation';
+import { patchConsole } from './logger/console-patch';
+
+patchConsole();
```

Comment:
I changed this to be more explicit and give us more control on where we want to apply the patch.